ext-src/services/RequestService.ts

```diff
@@ -16,7 +16,7 @@ import { BaseRequestService } from "./BaseRequestService";
  * limitations under the License.
  */
 export interface RequestService extends BaseRequestService {
-  getApplicationId(applicationPublicId: string): Promise<any>;
+  getApplicationId(applicationPublicId: string): Promise<string>;
```

Comment:
Lil tiny thing to make this app better, try and switch away from using `any` all the time and actually use TypeScript :)